docs/src/modules/components/AppContent.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import React from 'react';
+import * as React from 'react';
```

Comment:
Sounds good then for the source code 👍. I haven't looked at the demos, but I don't think that they should be flowtyped covered as used in an unknown environnement. I have been using flow weak so far.